include/brickengine/collision_return_values.hpp

```diff
@@ -0,0 +1,11 @@
+struct collisionReturnValues{
+    double spaceLeft;
+    int objectId;
+    collisionReturnValues(double spaceLeft, int objectId) : spaceLeft(spaceLeft), objectId(objectId){}
+};
+
+struct triggerReturnValues{
```

Comment:
Should have upercase Names, constructor needs to be on top and a space between the end of the initializers and _{}_